lib/puppet/transaction/report.rb

```diff
@@ -260,8 +258,6 @@ def initialize_from_hash(data)
 
     if data['server_used']
       @server_used = data['server_used']
-    elsif data['master_used']
-      @server_used = data['master_used']
```

Comment:
Sorry I missed this, I think we should always support reading the old `master_used` value, and setting `@server_used`. When writing a report, we should always set `server_used`.